plugins/google-oauth2-config.conf

```diff
@@ -41,3 +41,15 @@
 #   pass,\
 #   nolog,\
 #   setvar:'tx.google-oauth2-plugin_enabled=0'"
+
+# Here you can whitelist additional GET parameters which you are sending
+# using callback URL and are not standard to Google OAuth2. Use this syntax:
+# setvar:'tx.google-oauth2-plugin_whitelisted_parameters=/param1/ /param2/ /param3/'
+SecAction \
+ "id:9505020,\
+  phase:1,\
+  nolog,\
+  pass,\
+  t:none,\
+  ver:'google-oauth2-plugin/1.0.0',\
+  setvar:'tx.google-oauth2-plugin_whitelisted_parameters='"
```

Comment:
Apparently it is illegal to use `setvar` with a variable without a value. You'll probably have to comment the rule.

Reply:
It is working fine on Apache, i tested it before sending PR (and retested now). Is it something which is nginx related?

Reply:
Yes, looks like it only affects nginx.